app/assets/stylesheets/modules/subnavbar.scss

```diff
@@ -103,4 +115,20 @@
       white-space: normal;
     }
   }
+
+  // Responsive fixes for COVID-19 status
```

Comment:
These changes should get backed out once COVID-19 status is no longer in the menu.
### Before
![responsive-issue](https://user-images.githubusercontent.com/5402927/94325533-bb6be400-ff53-11ea-94df-fef3278ceffc.gif)

### After
![fixed-responsive](https://user-images.githubusercontent.com/5402927/94325538-bf980180-ff53-11ea-8019-7423799c07a9.gif)